oldlx/Trees/Patterns_Evy.java

```diff
@@ -0,0 +1,69 @@
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
+
+import heronarts.lx.LX;
+import heronarts.lx.LXChannel;
+import heronarts.lx.LXUtils;
+import heronarts.lx.color.LXColor;
+import heronarts.lx.modulator.Accelerator;
+import heronarts.lx.modulator.Click;
+import heronarts.lx.modulator.LinearEnvelope;
+import heronarts.lx.modulator.SawLFO;
+import heronarts.lx.modulator.SinLFO;
+import heronarts.lx.modulator.TriangleLFO;
+import heronarts.lx.modulator.QuadraticEnvelope;
+import heronarts.lx.parameter.BasicParameter;
+import heronarts.lx.parameter.BooleanParameter;
+import heronarts.lx.parameter.DiscreteParameter;
+import heronarts.lx.parameter.LXParameter;
+
+import toxi.geom.Vec2D;
+import org.apache.commons.lang3.ArrayUtils;
+import toxi.math.noise.PerlinNoise;
+import toxi.math.noise.SimplexNoise;
+
+
+/*
+ * A circle expanding in and out from the origin, inspired the "breath" pattern and misc breathing exercise graphics i've seen
+ */
+class CircleBreath extends TSPattern {
+  static final float MAX_R = 1480;
```

Comment:
aside: I think this would be a nice variable to have in a constants file somewhere - both Charlie S and I used it, and I imagine it could be useful for other patterns, too